database/base.py

```diff
@@ -38,6 +38,14 @@ class Base(DeclarativeBase):
 
 metadata = Base.metadata
 
+convention = {
```

Comment:
This is good to have, but it is not used anywhere yet?

Reply:
Oh, Shouldn't be in the commit, leftovers from my tests.